src/openfermion/hamiltonians/_general_hubbard.py

```diff
@@ -177,6 +184,8 @@ def __init__(self, lattice,
             potential_parameters (Iterable[Tuple[int, Number]], optional): The
                 potential parameters.
             magnetic_field (Number, optional): The magnetic field. Default is 0.
```

Comment:
Change this from Number to float

Reply:
What about `Real`?

Reply:
That's not a type ...

Reply:
https://docs.python.org/3/library/numbers.html#numbers.Real

Reply:
Let's just do the normal thing and call it a float.

Reply:
... but it's not important enough to block this.

Reply:
Sorry for letting this sit. I just changed it, because it doesn't really matter. (It's just in a docstring; if we were actually typechecking, I'd probably want to allow ints, but it seems that typing is a lot more informal in OpenFermion.)

Reply:
@kevinsung ready for re-review